tf2/CMakeLists.txt

```diff
@@ -104,7 +99,6 @@ if(BUILD_TESTING)
 endif()
 
 ament_export_dependencies(console_bridge geometry_msgs rcutils rosidl_runtime_cpp)
-ament_export_include_directories(include)
-ament_export_libraries(tf2)
```

Comment:
Is this going to break downstream consumers that rely on the old CMake variables?

Reply:
It would. I added back all the old-style CMake variable stuff in https://github.com/ros2/geometry2/pull/493/commits/92332fb40508dd6dde837b7ba6b9d1b2a4143d96